test/e2e/README.md

```diff
@@ -1,40 +1,50 @@
 # Zarf End-To-End Tests
 
-This directory holds all of our e2e tests that we use to verify Zarf functionality in an environment that replicates a live setting. The tests in this directory are automatically run against all the default K8s distros whenever a PR is opened against the repo. 
+This directory holds all of our e2e tests that we use to verify Zarf functionality in an environment that replicates a live setting. The tests in this directory are automatically run against several K8s distros whenever a PR is opened or updated.
 
-
-# Running Tests Locally
+## Running Tests Locally
 The tests in this directory are also able to be run locally!
 
-## Dependencies
+### Dependencies
 Running the tests locally have the same prerequisites as running and building Zarf:
  1. GoLang >= `1.16.x`
  2. Make
- 3. Docker 
-
+ 3. (for the `kind` and `k3d` options only) Docker
+ 4. (for the `k3s` cluster only) Linux with root privileges
 
 ### Existing K8s Cluster
-If you have a cluster already running, great! As long as your kubeconfig is accessible at `~/.kube/.config` or is exposed in your `$KUBECONFIG` environment variable, the e2e tests will be able to use your cluster to run your tests. This means that you can even test remote distros like EKS, AKS, and GKE. When the tests are completed, your cluster will still be accessible but note that since the tests are actively using your cluster it might be in a very different state.
+If you have a cluster already running, and use the env var `TESTDISTRO=provided`, the test suite will use the `KUBECONFIG` env var and the cluster that is currently configured as the active context. To make sure you are running in the right cluster, run something like `kubectl get nodes` with no optional flags and if the nodes that appear are the ones you expect, then the tests will use that cluster as well.
```

Comment:
This might be a good new issue to expose the distro detect stuff on the CLI.  It would be really clean for a `zarf tools what-distro` or something command to tell the user about their kubecontext + detected distro.

Reply:
Created #412